cmd/alertmanager/main.go

```diff
@@ -423,3 +433,10 @@ func mustHostname() string {
 	}
 	return hostname
 }
+
+func md5HashAsMetricValue(data []byte) float64 {
+	sum := md5.Sum(data)
+	var bytes []byte = sum[0:8]
+	bits := binary.LittleEndian.Uint64(bytes)
+	return math.Float64frombits(bits)
```

Comment:
48 bits sounds reasonable regarding the NaN problem.

> storing it as a value rather than bits

What do you mean by this?